moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -225,29 +254,44 @@ void JogCalcs::run(const ros::TimerEvent& timer_event)
   // Create new outgoing joint trajectory command message
   auto joint_trajectory = moveit::util::make_shared_from_pool<trajectory_msgs::JointTrajectory>();
 
-  // If the input command isn't stale, run calculations
-  if (!command_is_stale_)
+  // Prioritize cartesian jogging above joint jogging
+  // Only run commands if not stale and nonzero
+  if (have_nonzero_twist_stamped_ && !twist_command_is_stale_)
+  {
+    if (!cartesianJogCalcs(twist_stamped_, *joint_trajectory))
+    {
+      resetLowPassFilters(original_joint_state_);
+      return;
+    }
+  }
+  else if (have_nonzero_joint_jog_ && !joint_command_is_stale_)
   {
-    // Prioritize cartesian jogging above joint jogging
-    if (have_nonzero_twist_stamped_)
+    if (!jointJogCalcs(joint_jog_, *joint_trajectory))
```

Comment:
The name of `joint_jog_` should be more clear. Like, `joint_jog_cmd_`

Reply:
Agree, changed it to `joint_jog_cmd_`, and also changed the similar `twist_stamped_` to `twist_stamped_cmd_`, see latest commit